searchlib/src/vespa/searchlib/docstore/writeablefilechunk.cpp

```diff
@@ -484,11 +484,19 @@ void
 WriteableFileChunk::updateChunkInfo(const ProcessedChunkQ & chunks, const ChunkMetaV & cmetaV, size_t sz)
 {
     std::lock_guard guard(_lock);
+    uint32_t maxChunkId(0);
+    for (const auto & chunk : chunks) {
+        maxChunkId = std::max(chunk->getChunkId(), maxChunkId);
+    }
```

Comment:
Could this be done prior to the lock being taken?

Reply:
Yes, #20360